.github/workflows/validate-translation-files.yml

```diff
@@ -41,18 +47,29 @@ jobs:
           cache-dependency-path: |
             requirements/translations.txt
 
-      - name: Install requirements
-        if: steps.filter.outputs.pofiles == 'true'
+      - name: Install python requirements
+        if: steps.filter.outputs.changedtranslations == 'true'
         run: |
           python --version
           make translations_scripts_requirements
 
-      - name: Validate translation files
+      - name: Setup node
+        if: steps.filter.outputs.changedtranslations == 'true' && matrix.file-type == 'json'
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

Comment:
We've standardized on using `.nvmrc` files in most of the org repos to manage node versions. If we do that we can have this action use `node-version-file` ([docs](https://github.com/actions/setup-node?tab=readme-ov-file#usage)) instead.